[Please squash] Update sys-dm-database-backups-azure-sql-database.md (#9818)
MicrosoftDocs/sql-docs: docs/relational-databases/system-dynamic-management-views/sys-dm-database-backups-azure-sql-database.md

Returns information about backups of a database in an [!INCLUDE [ssazure-sqldb](../../includes/ssazure-sqldb.md)] server.

> [!NOTE]  

> > The `sys.dm_database_backups` DMV is currently in preview and is available for all Azure SQL Database service tiers except Hyperscale tier. Running this DMV in the Hyperscale tier returns unexpected results, because the Hyperscale tier is snapshot-based.

Reviewer: ```suggestion
> The `sys.dm_database_backups` DMV is currently in preview and is available for all Azure SQL Database service tiers except the Hyperscale tier. Since the Hyperscale service tier relies on snapshots for backups, running this DMV in the Hyperscale service tier returns unexpected results.
```

> > The `sys.dm_database_backups` DMV is currently in preview and is available for all Azure SQL Database service tiers except the Hyperscale tier. Since the Hyperscale service tier relies on snapshots for backups, running this DMV in the Hyperscale service tier returns unexpected results.

Reviewer 2: ```suggestion
> The `sys.dm_database_backups` DMV is currently in preview and is available for all Azure SQL Database service tiers except the Hyperscale tier. Since the Hyperscale service tier relies on snapshots for backups, running this DMV in the Hyperscale service tier returns no results.
```

> > Although the `sys.dm_database_backups` DMV is currently in preview and is available for all Azure SQL Database service tiers, the Hyperscale tier will not have correct content as it is snapshot based and in the snapshot you can see only Transaction log backup apart from the first Full Back Up.

Reviewer: ```suggestion
> The `sys.dm_database_backups` DMV is currently in preview and is available for all Azure SQL Database service tiers except Hyperscale tier. Running this DMV in the Hyperscale tier returns unexpected results, because the Hyperscale tier is snapshot-based.
```
